spec/services/github_client/user_spec.rb

```diff
@@ -1,27 +1,27 @@
 require 'rails_helper'
 
 RSpec.describe GithubClient::User do
-  describe '#repositories' do
+  describe '#pull_request_events' do
```

Comment:
The method got considerably more complex now. Could we add some more tests? (The time restriction is not tested, for example)